2020-01-Winter/2_string_manipulation/README.md

```diff
@@ -45,21 +45,38 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. GROUP ANAGRAMS:
 
-Source: TODO :bug:
+Source:  [LeetCode](https://leetcode.com/problems/group-anagrams/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given an array of strings, group anagrams together.
+
+All inputs will be in lowercase.
+
+The order of your ouput does not matter.
 
 #### Example Input
 
-If the problem is simple enough, remove this section. TODO :bug:
+```
+Input = ["eat","tea","tan","ate","nat","bat"]
+
+Output = [
+  ["ate","eat","tea"],
+  ["nat","tan"],
+  ["bat"]
+]
+```
 
 #### Function Signature
 
-TODO :bug:
+Java:
+```
```

Comment:
Add language name to opening backtick block to get syntax highlighting for your code, if you like
```suggestion
```java
```